ci/ray_ci/container.py

```diff
@@ -67,6 +65,20 @@ def _docker_run_bash_script(script: str) -> str:
     return _get_docker_run_command() + ["/bin/bash", "-ice", script]
 
 
+def shard_tests(test_targets: List[str], shard_count: int, shard_id: int) -> List[str]:
+    """
+    Shard tests into N shards and return the shard corresponding to shard_id
+    """
+    return (
+        run_command(
+            "python ci/run/bazel_sharding/bazel_sharding.py "
```

Comment:
use a list of args? and use `sys.executable` to launch python? which is more reliable in a bazelized python script.

Reply:
also, this would potentially drag in many dependencies?

can we just move that script into `ray_ci`, bazelize it into a python library, and just call the library function?